frontend/src/metabase/home/components/EmbedHomepage/tests/common.unit.spec.tsx

```diff
@@ -169,14 +169,10 @@ describe("EmbedHomepage (OSS)", () => {
       const body = await lastCall?.request?.json();
       expect(body).toEqual({ value: "dismiss-run-into-issues" });
 
-      const feedbackBody = await getLastFeedbackCall()?.request?.json();
-
-      expect(feedbackBody).toEqual({
-        source: "embedding-homepage-dismiss",
-      });
-
       await waitForElementToBeRemoved(() => queryFeedbackModal());
 
+      expect(getLastFeedbackCall()).toBeUndefined();
```

Comment:
I moved it after the waitFor to make sure the request is not just delayed.
I checked and this test fails without the fix

Reply:
I think adding in reasoning like this to the code comment itself is quite useful for later editors, as it's not immediately clear to me what `expect(getLastFeedbackCall()).toBeUndefined()` is for, or supposed to do

Reply:
Thanks, added an explanation